ballerina/Ballerina.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 org = "ballerina"
 name = "xslt"
-version = "2.2.3"
+version = "2.3.0"
```

Comment:
distribution still points to 2201.0.4

Reply:
This file gets updated from a template. There is no point updating here, since the next build will ovveride

Reply:
Yes, the template was updated in [78846df](https://github.com/ballerina-platform/module-ballerina-xslt/pull/424/commits/78846dfac0f2451b5a5e41e7eb1244a5ebfffa67)